frontend/rolecall/src/app/cast/cast-editor-v2.component.ts

```diff
@@ -20,6 +27,7 @@ export class CastEditorV2 implements OnInit {
   allCasts: Cast[] = [];
   filteredCasts: Cast[] = [];
   allPieces: Piece[] = [];
+  popupList: PopupData[] = [];
```

Comment:
Let's call it something that describes what it represents rather than generic "popupData". How about:
`pieceMenuItems: PieceMenuItem[] = [];`

Reply:
Done.